CIP-?/README.md

```diff
@@ -0,0 +1,297 @@
+
+ - A governance metadata document MAY include a `@context` field.
+ - A governance metadata document MAY include a `@type` field, referring to a specific type of document from the included `@context`s. 
+ - The `@context` field, if included, MUST be a valid JSON-LD @context field; the basics are described below. 
+   - It MAY be a string, in which case it MUST be the IRI of a jsonld document describing the shared vocabulary to assume when interpreting the document.
+   - It MAY be an object, where each key refers to a property name, and the value refers to the IRI describing how that field should be interpreted.
+   - It MAY be an array, including multiple contexts, which are processed in order.
+   - For a full understanding of the @context field, refer to the [JSON-LD specification](https://www.w3.org/TR/json-ld/).
+ - If the metadata document is missing the `@context` field, it will be assumed to refer to [./cip-?.common.jsonld]
+ - Future CIPs may standardize common contexts, and SHOULD attempt to reuse common terminology and SHOULD avoid naming collisions.
+ - Tool authors MAY choose which contexts to support, but MUST make a best effort to display the metadata in the presence of unrecognized context, up to and including gracefully falling back to a raw display of the JSON document.
+
+### Governance Transaction Types
+
+In CIP-1694 (and likely any alternative or future evolution of it), there are a number of certificates that can be attached to a transaction pertaining to governance; each of these is equipped with an "anchor", which can refer to an external document. This CIP provides the following JSON-LD schemas intended to represent a fairly minimal set of properties and types for each type of transaction:
+
+ - [Common Definitions](./cip-?.common.jsonld)
+ - [Governance Action](./cip-?.action.jsonld)
+ - [DRep (Voter) Registration](./cip-?.registration.jsonld)
+ - [DRep (Voter) Retirement](./cip-?.retirement.jsonld)
+ - [DRep (Voter) Vote](./cip-?.vote.jsonld)
+ - [DRep (Voter) Delegation](./cip-?.delegation.jsonld)
+ - [Constitutional Committee Vote](./cip-?.cc-vote.jsonld)
+
+<!-- The above schemas aren't written yet; this is one of the areas I could use some help in, and I didn't want that to be a further impediment to me getting this draft out the door, as it's taken me long enough as it is -->
+
+Additionally, someone may wish to augment a previous piece of metadata with new information, divorced from the transaction that initially published it; this may be useful, for example, to provide additional arguments in favor of or against a proposal, provide translations to other languages, provide a layman's explanation of a highly technical document, etc.
+
+To this end, this CIP reserves [metadatum label 1694](../CIP-0010/README.md) for publishing these kinds of augments on-chain. This CIP also provides a special JSON-LD schema specifically for these documents:
```

Comment:
This is an interesting idea! although I am not sure how tools should handle this? how a later metadata transaction will able to be linked to a past governance action?

Reply:
The idea is just to publish a transaction with the relevant addendum, and one of the fields uniquely identifies which document(s) it's responding to. For example, you could provide a translation for the metadata attached to proposal `1ef...#0`